Dockerfile

```diff
@@ -111,45 +117,8 @@ ENV LD_PRELOAD /usr/lib/preloadable_libiconv.so php
 # Copy TTRSS and plugins
 COPY --from=builder /var/www /var/www
 
-# Install GNU libc (aka glibc) and set C.UTF-8 locale as default.
-# https://github.com/Docker-Hub-frolvlad/docker-alpine-glibc/blob/master/Dockerfile
```

Comment:
I've tried `cache_starred_images` without adding glibc and it no longer causes error now.
So it should be safe to assume it won't cause #118 again.